app/src/routes/shared/shared.vue

```diff
@@ -132,6 +133,10 @@ async function authenticate() {
 		authenticating.value = false;
 	}
 }
+
+useHead({
+	title: title,
+});
```

Comment:
Can be shortened to
```suggestion
useHead({ title });
```

or

```suggestion
useHead({
	title
});
```